src/libsystemd/sd-bus/bus-socket.c

```diff
@@ -885,6 +885,11 @@ int bus_socket_connect(sd_bus *b) {
                 assert(b->output_fd < 0);
                 assert(b->sockaddr.sa.sa_family != AF_UNSPEC);
 
+                _cleanup_free_ char *pretty = NULL;
+                (void) sockaddr_pretty(&b->sockaddr.sa, b->sockaddr_size, false, true, &pretty);
+                log_debug("sd-bus: starting bus%s%s by connecting to %s...",
+                          b->description ? " " : "", strempty(b->description), strnull(pretty));
```

Comment:
please enclose this in `if (DEBUG_LOGGING)` since it does extra syscall work when debug logging is on